src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/CompareSymbolsCorrectlyTests.cs

```diff
@@ -1248,9 +1248,156 @@ public int GetHashCode(object o1, object o2)
             }.RunAsync();
         }
 
+        [Fact]
```

Comment:
Could you add the various vbnet parts?

Reply:
Will fix, probably in few days :(

Feel free to re-ping me if I forgot.